.circleci/config.yml

```diff
@@ -1,9 +1,32 @@
 version: 2
 
 jobs:
+  build-linux-ghc-bindist:
+    docker:
+      - image: debian
+    working_directory: ~/rules_haskell
+    resource_class: large
+    steps:
+      - checkout
+      - run:
+          name: Install Bazel
+          command: |
+            apt-get update
+            apt-get install -y curl gnupg golang make libgmp3-dev
+            echo "deb [arch=amd64] http://storage.googleapis.com/bazel-apt stable jdk1.8" | tee /etc/apt/sources.list.d/bazel.list
+            curl https://bazel.build/bazel-release.pub.gpg | apt-key add -
+            apt-get update
+            apt-get install -y bazel
+      - run:
+          name: Build tests
```

Comment:
Don't copy the test script from Windows (which is a WIP target with only few tests passing)! Copy it from the other build jobs in this very file (Linux and Darwin).

Reply:
I have not managed to build any other tests using the `rules_haskell` master using the bindist toolchain. I though it was an expected behaviour as we get most of the tests hackage deps using the nixpkgs.


See the error logs at:

https://circleci.com/gh/tweag/rules_haskell/3646